StockBugFix/StockBugsPatches.cs

```diff
@@ -452,5 +453,61 @@ public static class SolidTransferArm_OnSpawn_Patch {
 					storage.SetDefaultStoredItemModifiers(Storage.StandardSealedStorage);
 			}
 		}
+
+		/// <summary>
+		/// Applied to SpaceHeater to fix Tepidizer target temperature area being too large.
+		/// </summary>
+		[HarmonyPatch(typeof(SpaceHeater), "MonitorHeating")]
+		public static class SpaceHeater_MonitorHeating_Patch {
+			/// <summary>
+			/// Transpiles MonitorHeating to replace the GetNonSolidCells call.
+			/// </summary>
+			internal static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> method) {
+				List<CodeInstruction> instructionList = method.ToList();
+				MethodInfo targetMethod = typeof(GameUtil).GetMethodSafe("GetNonSolidCells",
+					true, typeof(int), typeof(int), typeof(List<int>));
+				int targetIndex = -1;
+				for (int i = 0; i < instructionList.Count; i++) {
+					CodeInstruction instruction = instructionList[i];
+					if (instruction.opcode == OpCodes.Call && instruction.operand != null &&
+						instruction.operand.Equals(targetMethod)) {
+						targetIndex = i;
+						break;
+					}
+				}
+				if (targetIndex == -1) {
+#if DEBUG
+					PUtil.LogError("Target method GetNonSolidCells not found.");
```

Comment:
`LogError` will blackhole the game, which is not an ideal thing to do in a `DEBUG`-only code section. Consider downgrading to a `Warning` and removing `#if DEBUG`.

Reply:
Fixed